its/projects/CSharp.SDK.7.0/AspNetCoreMvc/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+﻿{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:24851",
+      "sslPort": 44304
+    }
+  },
+  "profiles": {
```

Comment:
[simplicity] I think you can only keep one profile?

and if you keep the `https` profile, you can drop the `iisSettings` above

(less is more, we should keep only some minimal functionality for what static analysis is concerned - these are runtime settings as far as I can tell)